Client/Teztez/Flows/Common/ActivitiesIntro/Store/ActivitiesIntroStore.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ActivitiesIntroStore.swift
+//  Teztez
+//
+//  Created by Adlet on 4/23/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+// TODO: - Remove
+struct ActivitiesIntro {
+    let title: String
+    let image: String
+    let infoTitle: String
+    let infoDescr: String
+    let infoImage: String
+    let usageTitle: String
+    let usageDescr: String
+    let usageImage: String
+}
+
+// swiftlint:disable all
+final class ActivitiesIntroStore {
```

Comment:
You did everything seemingly right here :)

But, I think we can just keep it simple and just have `ActivitiesIntroViewController` and `ActivitiesIntroViewModel`.
Store is not really necessary here I think, let's keep it simple :)